src/main/java/com/project/videotecha/model/Reservation.java

```diff
@@ -0,0 +1,66 @@
+package com.project.videotecha.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+
+@Entity
+public class Reservation {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(optional = false)
+    @JoinColumn(nullable = false)
+    private Projection projection;
+    @ManyToOne(optional = false)
+    @JoinColumn(nullable = false)
+    private User user;
+    private Integer numberOfSeats;
+    private Boolean canceled = false;
```

Comment:
Why Boolean and not boolean here? Can this value be absent? If not, then primitive is fine